aptos-move/framework/aptos-framework/sources/configs/version.move

```diff
@@ -52,11 +54,10 @@ module aptos_framework::version {
 
     /// Used in on-chain governances to update the major version for the next epoch.
     /// Example usage:
+    /// - `aptos_framework::version::set_for_next_epoch(&framework_signer, new_version);`
+    /// - `aptos_framework::aptos_governance::reconfigure(&framework_signer);`
     /// ```
```

Comment:
You're mixing two types of blocks.  Either:

```suggestion
    /// - `aptos_framework::version::set_for_next_epoch(&framework_signer, new_version);`
    /// - `aptos_framework::aptos_governance::reconfigure(&framework_signer);`
```

OR

````suggestion
    /// ```
    /// aptos_framework::version::set_for_next_epoch(&framework_signer, new_version);
    /// aptos_framework::aptos_governance::reconfigure(&framework_signer);
    /// ```
````